### ISSUE TITLE
Notices extend beyond the RHS

### ISSUE THREAD

**Issue by kenzierocks** [marumari/textual-theme-Equinox#12]
![notices extend far](https://i.imgur.com/THIvPZb.png)

**Comment by kenzierocks** [comment]
This triggers it for me:
```
/fakerawdata :kenzierocks!~kenzieroc@this.is.kenzierocks.me NOTICE #textual-testing :a really long test is long for the sake of overflowing the right hand side of textuals fields. this is very long. following with many As. AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA
```

**Comment by marumari** [comment]
Fixed in 75202018726c67fb4d05142b44227833bc7d76a1.
